apps/package.json

```diff
@@ -50,20 +50,7 @@
     "@babel/polyfill": "^7.12.1",
```

Comment:
Starting here we're in the devDependencies section of package.json, red here means I decided something currently a devDep should actually be a dep